source/D0323r5.bs

```diff
@@ -633,30 +661,36 @@ otherwise if `bool(*this)` and `!bool(rhs)`,
 
   * if `T` is `void`
    
-    * initializes `unexpect` as if direct-non-list-initializing an object of type `unexpected<E>` with `unexpected(rhs.error())`. Either
+    * initializes `unexpect` as if direct-non-list-initializing an object of type
+`unexpected<E>` with `unexpected(rhs.error())`. Either
```

Comment:
Doesn't this line wrap break the markdown format that bikeshed expects? I don't think you'll get a list from this anymore. Same applies to other places below.

Reply:
I was using Macdown and there is no issue on the visualizer. If you find an issue with the tools you use to generate the html document I'll rollback these changes.

I wanted just to follow your advice about splitting the lines too long.

Reply:
Could you resolve the branch conflict that GitHub complains about? I'll try it out after.

Reply:
I did it online, but maybe this is not enough. The result is my own file.
Maybe I need to merge your master branch in my won master branch.
I'll try it if I have some time today.

Reply:
Here it is what I've done

```
imac-de-vicente-botet-escriba:jf_papers viboes$ git fetch upstream
From https://github.com/jfbastien/papers
   9cb5a45..7790e4a  master     -> upstream/master
imac-de-vicente-botet-escriba:jf_papers viboes$ git checkout master
Already on 'master'
Your branch is up-to-date with 'origin/master'.
imac-de-vicente-botet-escriba:jf_papers viboes$ git merge upstream/master
Already up-to-date.
imac-de-vicente-botet-escriba:jf_papers viboes$ git status
On branch master
Your branch is up-to-date with 'origin/master'.

```

What I should do to merge your repo?

Reply:
You also need to then push your changes here.

My normal workflow is:

```
git checkout master
git checkout -b awesome_work
# work work work
git commit -am "Awesome work"
git checkout master
git pull origin master
git checkout awesome_work
git rebase master
git push my_repo awesome_work
```

Then you go to GitHub, and on this repo it'll tell you "your fork has changes, do you want to create a pull request with them?" You do so, and from then on all pushes to my_repo's awesome_branch will appear in that one pull request.